### PR TITLE
add remote access instructions

### PR DESCRIPTION
**Pull request by Riscky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,4 +15,27 @@ This repository contains an Ansible playbook used to set up a Raspberry Pi
...
+User solisnummer
+
+Host tunnel-radio
+User pi
+Hostname stickykamer.mynetgear.com
+Port 2200
+ProxyJump gemini.science.uu.nl
+```
+
+`ssh -p 2200 pi@stickykamer.mynetgear.com`
```
Comment (Mstiekema) [inline README.md]: This line should be:
```md
3. Run `ssh -p 2200 pi@tunnel-radio`
```

**Review by Mstiekema — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Riscky) [reply README.md]: thats only if your inside the sticky room right?

Comment (Mstiekema) [reply README.md]: Nope, you declare the tunnel itself in the config as `tunnel-radio`. In the Sticky room you use `pi@stickykamer.mynetgear.com`

**Review by Mstiekema — APPROVED** [review]
LGTM

**Pull request merged**